### PR TITLE
R4_Diagnostic_Report_Functional_Overview_updated

### PR DESCRIPTION
**Pull request by JagatNayak**
Description
----

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

**Comment by JagatNayak** [comment]
Screenshot 2022-10-21 at 10 22 58 AM

**Comment by JagatNayak** [comment]
Screenshot 2022-12-08 at 10 16 19 AM
Screenshot 2022-12-08 at 10 16 26 AM
Screenshot 2022-12-08 at 10 16 36 AM
Screenshot 2022-12-08 at 10 16 50 AM
Screenshot 2022-12-08 at 10 16 57 AM
Screenshot 2022-12-08 at 10 17 04 AM
Screenshot 2022-12-08 at 10 17 16 AM
Screenshot 2022-12-08 at 10 17 25 AM

File: content/millennium/r4/clinical/diagnostics/diagnostic-report.md [hunk 1]
```diff
@@ -69,7 +63,7 @@ The following fields are returned if valued:
  `patient`        | This or `_id`     | [`reference`] | The subject of the report if a patient. Example: `12345`
  `encounter`      | N                 | [`reference`] | The Encounter when the order was made. Must represent an Encounter resource. May include a single or comma separated list of references. Example: `encounter=1621910`
  `date`           | N                 | [`date`]      | Date range into which the diagnostic report falls (effectiveDateTime). Either 1 or 2 date/time can be given. Example: `date=lt2017-01-5`
- `_count`         | N                 | [`number`]    | The maximum number of results to return per page. Example: `50`
+ `_count`         | N                 | [`number`]    | The maximum number of results to return.Default maximum supported = 100.
```
Comment (ashu-bdvt) [inline content/millennium/r4/clinical/diagnostics/diagnostic-report.md:66]:
```suggestion
 `_count`         | N                 | [`number`]    | The maximum number of results to return. Default maximum supported = 100.
```

**Review by ashu-bdvt — APPROVED** [review]

**Comment by JagatNayak** [comment]
image

**Comment by JagatNayak** [comment]
image

**Review by NimishaMN — APPROVED** [review]

**Review by ashu-bdvt — APPROVED** [review]

**Comment by JagatNayak** [comment]
Screenshot 2022-12-08 at 12 57 25 PM
Screenshot 2022-12-08 at 12 57 33 PM
Screenshot 2022-12-08 at 12 57 40 PM
Screenshot 2022-12-08 at 12 57 53 PM
Screenshot 2022-12-08 at 12 58 00 PM
Screenshot 2022-12-08 at 12 58 08 PM
Screenshot 2022-12-08 at 12 58 17 PM
Screenshot 2022-12-08 at 12 58 32 PM

**Comment by JagatNayak** [comment]
Screenshot 2022-12-15 at 10 03 46 AM
Screenshot 2022-12-15 at 10 03 59 AM

**Comment by vinuthnag** [comment]
changes after publishing

image

image

**Pull request merged**